platforms/android/app/src/main/java/com/tns/NativeScriptActivity.java

```diff
@@ -93,4 +100,84 @@ public int hashCode__super() {
   public boolean equals__super(java.lang.Object other) {
     return super.equals(other);
   }
+
+  Messenger mService = null;
```

Comment:
The logic below will be ideally injected only in debug builds. It can be either injected during class/method generation in Static Binding generator or extracted in separate class and try to just inject 1-2 lines using the class.

Reply:
The extacted class can just have a hollow implementation for release.